src/components/SignIn.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./signin.css";
```

Comment:
Don't use style sheets. Mainly rely on styled components and put re-usable styles in GlobalStyles

Reply:
How to write styles which will applicable to only certain pages that will not get reused

Reply:
How to write styles which will applicable to only certain pages that will not get reused ?